build.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
```

Comment:
I don't think this is the latest version of this file. see the templates in universe